source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,111 @@
+.. _telemetry-enable-disable:
+
+Enable and Disable Telemetry in Clear Linux
+###########################################
+
+|CLOSIA| includes a telemetry solution as part of the OS that records events
+of interest and reports them back to the development team via the telemetrics
+daemon, :command:`telemd`.  This functionality is maintained in the
+``telemetrics`` software bundle.
+
+.. Note::
+	The telemetrics functionality adheres to `Intel’s privacy policies`_
+	regarding the collection and use of Personally Identifiable Information
+	(PII) and is Open Source. Specifically, no intentionally identifiable
+	information about the user or system owner is collected.
+
+End users may enable or disable the telemetry component of |CL| or even
+redirect where records go if they wish to collect records for themselves.
+
+Install the telemetry software bundle
+*************************************
+
+During the initial installation of |CL| you are requested to join the
+Stability Enhancement Program and allow |CLOSIA| to collect anonymous reports

```

Comment:
Please do not use capital letters for emphasis. This should read: stability enhancement program ...

Reply:
That is the name of the menu during the installer, caps included.  I'll make them lower case here.